src/components/Dashboard/index.tsx

```diff
@@ -47,15 +47,25 @@ const Index: React.FunctionComponent = () => {
           Authorization: `Bearer ${session.accessToken}`,
           Accept: 'application/vnd.github.v3+json'
         };
+  
```

Comment:
@Sunnidhya I think it's been fixed through another PR https://github.com/instructlab/ui/pull/326/files (just testing it with the master branch). If user login through admin/pass there won't be any session token available, so in that case we just need to set the loading to false.